common/lib/capa/capa/responsetypes.py

```diff
@@ -1597,9 +1597,8 @@ def get_score(self, student_answers):
             student_float = evaluator({}, {}, student_answer)
         except UndefinedVariable as undef_var:
             raise StudentInputError(
-                _(u"You may not use variables ({bad_variables}) in numerical problems.").format(
-                    bad_variables=undef_var.message,
-                )
+                _(u"Answers can include numerals, operation signs, and a few specific characters, "
```

Comment:
Since we have the exact same string twice, can you make it a variable? Or does that cause problems because it can't be translated at that level, and therefore it wouldn't get scraped if we just store the English version as a variable?